src/components/NewBookForm.css

```diff
@@ -0,0 +1,17 @@
+.add-book-form {
+  display: flex;
+  gap: 10px;
+  margin-left: 3%;
+}
+
+.form-btn {
+  color: #fff;
+  background-color: #0290ff;
+  border: none;
+  border-radius: 3px;
+  padding: 10px;
+}
+
+.form-input {
+  border: none;
+}
```

Comment:
Kindly note that as part of the project requirement, your project should be as close as possible to the design given. Please use the necessary styling to ensure that your design is a close replica of the design given 😄 

Your design :
![image](https://github.com/PedroH-Neves/book-store-react/assets/20537712/c5367442-33b6-4658-a352-dc72163f8978)

Design given:
![image](https://github.com/PedroH-Neves/book-store-react/assets/20537712/a691fb6b-b621-4c58-8087-c78f6e5c6cea)

Reply:
As suggested previously, your design should be close to the design given. kindly add some margin between the book items details, so as to ensure that both the book details and the book form contains the ful screen just as is given in the design 👍 

Yur design :

![image](https://github.com/PedroH-Neves/book-store-react/assets/20537712/f21165e0-7953-4f72-8880-2fe1b0cc295c)

Design given :
![image](https://github.com/PedroH-Neves/book-store-react/assets/20537712/e116a1bb-e60d-4fe9-9bb9-b11897988c0e)

Reply:
{Optional] Please as suggested previously, kindly ensure that the buttons are in the same line vertically and also use the same button size for all the buttons, to ensure consistency and to ensure your design has a good UI

Your design :

![image](https://github.com/PedroH-Neves/book-store-react/assets/20537712/615b8f17-b04b-4a94-acc8-1bf09fa2d093)